core/impl/collection_manager.cxx

```diff
@@ -23,6 +23,8 @@
 #include "core/operations/management/scope_create.hxx"
 #include "core/operations/management/scope_drop.hxx"
 #include "core/operations/management/scope_get_all.hxx"
+
+#include "core/impl/error.hxx"
 #include "internal_manager_error_context.hxx"
```

Comment:
Is this include needed?

Reply:
Should we be able to completely remove `internal_manager_error_context` now?

Reply:
I think the include is needed for the core::impl::make_error methods.

Yes, let's remove the internal_manager_error_context and the underlying manager_error_context too. We should do this for the other operations which aren't used in the core too, but we can do that outside of this PR.

Reply:
Yes, makes sense. I was referring to the "internal_manager_error_context.hxx" include above